instrumentation/servlet/servlet-javax-common/javaagent/servlet-javax-common-javaagent.gradle

```diff
@@ -0,0 +1,30 @@
+apply from: "$rootDir/gradle/instrumentation.gradle"
+
+muzzle {
+  pass {
+    group = "javax.servlet"
+    module = "servlet-api"
+    versions = "(0,)"
+  }
+  pass {
+    group = "javax.servlet"
+    module = 'javax.servlet-api'
+    versions = "[3.0,)"
```

Comment:
Should this be 2.2 or 2.3?

Reply:
Above block handles that - Servlet 2 API uses `servlet-api` module name whereas Servlet 3 API uses `javax.servlet-api` module name.